server/graphql/v2/mutation/CreateCollectiveMutations.js

```diff
@@ -0,0 +1,130 @@
+import { GraphQLNonNull, GraphQLBoolean } from 'graphql';
+import { get, pick } from 'lodash';
+
+import { Collective } from '../object/Collective';
+import { CollectiveCreate } from '../input/CollectiveCreate';
+import { AccountInput, fetchAccountWithInput } from '../input/AccountInput';
+
+import * as errors from '../../errors';
+import models from '../../../models';
+import roles from '../../../constants/roles';
+import activities from '../../../constants/activities';
+import * as github from '../../../lib/github';
+import { purgeCacheForPage } from '../../../lib/cloudflare';
+import { defaultHostCollective } from '../../../lib/utils';
+
+const DEFAULT_COLLECTIVE_SETTINGS = {
+  features: { conversations: true },
+};
+
+async function createCollective(_, args, req) {
+  let shouldAutomaticallyApprove = false;
+
+  const { remoteUser, loaders } = req;
+
+  if (!remoteUser) {
+    throw new errors.Unauthorized({
+      message: 'You need to be logged in to create a collective',
+    });
+  }
+
+  const collectiveData = {
+    ...pick(args.collective, ['name', 'slug', 'description', 'tags']),
```

Comment:
So we will only need to input name, slug, desc, and tags from the form for the Collective to be created? (Plus the info from the remote user, etc...)